ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/functions/[slug]/(dashboard)/FunctionRunsChart.tsx

```diff
@@ -99,7 +99,7 @@ export default function FunctionRunsChart({
 
   return (
     <StackedBarChart
-      title="Function Runs"
+      title="Ended Function Runs"
```

Comment:
We don't need to change the title of this chart, they key point of confusion with this chart is not whether they have "ended" or not, it's the x-axis being the run start time and not the end time, which we need to change with the query. The current chart shows by start time, which means that amounts in the past can increase which is non-intuitive - it makes the chart look like it's broken when there is a backlog. If we make the change to the query, I believe it's all fixed.

The legend says "Completed" "Failed" and "Canceled" which are all past-tense, so "Ended" is assumed here.

Reply:
Makes sense. I reverted that commit to only leaving the View All Logs button change.